scripts/prepare-release.js

```diff
@@ -332,6 +333,55 @@ async function updateBackstageReleaseVersion(repo, type) {
   );
 }
 
+/**
+ * Ensures that the changesets include a version bump of create-app otherwise
+ * generates a new patch changeset for create-app.
+ */
+async function ensureCreateAppChangeset() {
+  process.chdir(path.resolve(__dirname, '../.changeset'));
```

Comment:
Let's avoid this, it mutates global state that is tricky to keep track of in this script